### DISCUSSION TITLE
ResizableTableContainer induces systematic horizontal scrollbar?

### DISCUSSION THREAD

**Discussion by janvorwerk** [adobe/react-spectrum#5906] [Q&A] [answered]
Hello gents

When you look at the storybook of the React-Apria table:
https://react-spectrum.adobe.com/react-aria-tailwind-starter/index.html?path=/docs/table--docs

you can see that it gets an unexpected horizontal scrollbar. I can get rid of the vertical scrollbar by removing the `max-h-[280px]` but no matter what I do, I don't manage to get rid of the horizontal one.

<img width="592" alt="image" src="https://github.com/adobe/react-spectrum/assets/139635115/0ae2681e-0fd8-495d-b69c-9032de3794cd">

Is this by design/spec?

I get the impression that the JS that computes the width is missing some borders/padding/.... ? so that, by default, the computed size is just a few px too wide.

Did I get something wrong?

Thanks

**Comment by snowystinger** [comment, marked as answer]
These are a starting point/example and it could definitely be better. It looks like it's coming from the resize handlers, they stick out a little into the next column, or in the case of the last column, it sticks out beyond the table. The reason for the extra padding on the handle is to increase the hit area for touch or users who need more affordance. The last one could be changed to have more padding on the inline-start and none on the inline-end possibly.

**Reply by janvorwerk** [reply]
Thanks for your answer!
Something very weird happened: that scrollbar has vanished overnight! 🤔 Both on the storybook and on my app!
Anyway, thank you again.
